x-pack/platform/plugins/shared/ml/public/application/jobs/jobs_list/components/jobs_list/jobs_list.js

```diff
@@ -422,6 +420,7 @@ export class JobsListUI extends Component {
           className={`jobs-list-table ${selectedJobsClass}`}
           items={pageOfItems}
           columns={columns}
+          tableLayout="auto"
```

Comment:
A couple of issues I've seen with setting `auto` for the table layout are:
- The ID column doesn't render well as the width decreases and the column gets less and less width
- As the width decreases the columns on the right go out of view and there isn't a horizontal scrollbar.


https://github.com/user-attachments/assets/9ffd453e-8680-4d32-afe6-a25e2b848da1

I tried removing `tableLayout="auto"` and setting `width: '50px'` on the icons column, which looks better I think, but worth giving it more of test:


https://github.com/user-attachments/assets/33296c4d-1fda-4d79-aa77-abb851638898